swatch-contracts/src/main/java/com/redhat/swatch/contract/service/OfferingProductTagLookupService.java

```diff
@@ -70,6 +70,7 @@ private static void processProductTagsBySku(
             .build();
 
     SubscriptionDefinition.getAllProductTags(lookupParams).forEach(productTags::addDataItem);
+    SubscriptionDefinition.pruneIncludedProducts(productTags.getData());
```

Comment:
I thought about including the "pruneIncludedProducts" to be used internally in "getAllProductTags", thought "getAllProductTags" is used in many places and I didn't want to introduce possible edge cases or break other functionality.

Reply:
:+1:, looking at `ProductNormalizer`, it gathers all possible product tags from various data sources and then calls pruneIncludedProducts, calling pruneIncludedProducts multiple times seems unnecessary.